pinot-controller/src/main/java/org/apache/pinot/controller/api/lineage/LineageManager.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.controller.api.lineage;
```

Comment:
Is there a better place to put this? 

RetentionManager is in `org.apache.pinot.controller.helix.core` and LineageManager is mainly utilized under `PinotHelixResourceManager`, which is also under `org.apache.pinot.controller.helix.core`.

Can you double check if this is a better candidate?

Reply:
Good point, will move this to `org.apache.pinot.controller.helix.core`